packages/phone-number-privacy/src/database/database.ts

```diff
@@ -0,0 +1,11 @@
+import { connectToDatabase } from '../config'
+
+const knex = connectToDatabase()
+
+export function getDatabase() {
+  return knex
+}
+
+export async function setSerializable() {
+  await knex.raw('BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE;')
```

Comment:
what does this do?

Reply:
handle concurrent requests (see discussion with @codyborn above)

"This level emulates serial transaction execution, as if transactions had been executed one after another, serially, rather than concurrently."

https://www.postgresql.org/docs/current/sql-set-transaction.html
^ i says it should send an error, although in practice I just see it fail on update and return a 0 (as in 0 rows updated)